### PR TITLE
Do not log status pings.

### PR DESCRIPTION
**Pull request by derrickburns**

### CODE REVIEW

File: service/middleware/access_log.go [hunk 1]
```diff
@@ -38,11 +39,24 @@ func NewAccessLog() (*AccessLog, error) {
 	return &AccessLog{}, nil
 }
 
+func (a *AccessLog) ignore(req *rest.Request) bool {
+	// ignore liveness and readiness probes
+	if strings.Compare(req.URL.RequestURI(), "/status") == 0 {
```
Comment (pazaan) [inline service/middleware/access_log.go]:
```suggestion
	if req.URL.RequestURI() == "/status") {
```

From the [go documentation](https://golang.org/pkg/strings/#Compare):
> Compare is included only for symmetry with package bytes. It is usually clearer and always faster to use the built-in string comparison operators ==, <, >, and so on

**Comment by pazaan** [comment]
@derrickburns Also backmerge from `develop` :)

File: service/middleware/access_log.go [hunk 2]
```diff
@@ -3,6 +3,7 @@ package middleware
 import (
 	"fmt"
 	"net"
+	"strings"
```
Comment (pazaan) [inline service/middleware/access_log.go]: `strings` is an unneeded import now.

**Comment by derrickburns** [comment]
[Quiet noisy log messages from platform services.](https://trello.com/c/BH1npNwU/639-quiet-noisy-log-messages-from-platform-services)

**Comment by derrickburns** [comment]
[Suppress noisy platform log messages.](https://trello.com/c/YbawaBoZ/643-suppress-noisy-platform-log-messages)

**Pull request merged**